languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -7811,6 +7811,67 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
                 <token postag='A.+' postag_regexp='yes'/>
                 <example>Deixei o local um pouco irritada.</example>
             </antipattern>
+
+            <rule id='PELO_QUAL_PELA_QUAL' name="Concordância Género: 'pelo' qual → 'pela'" default="temp_off">
```

Comment:
Let's avoid dialect-biased spelling in the rule name. Better: `Concordância: 'pelo' → 'pela'`